.qube.yml

```diff
@@ -0,0 +1,23 @@
+full_name: Sven Fillinger
+email: sven.fillinger@qbic.uni-tuebingen.de
+project_name: omero-portlet
+project_short_description: Omero-portlet. A imaging visualisation portlet for qPortal.
+version: 1.1.0
+domain: portlet
+language: groovy
+project_slug: omero-portlet
+project_slug_no_hyphen: omero_portlet
+template_version: '1.0.2 # <<QUBE_NO_BUMP>>'
+template_handle: portlet-groovy
+github_username: sven1103
+creator_github_username: sven1103
+is_github_repo: false
+is_repo_private: false
+is_github_orga: false
+copyright_holder: QBiC
```

Comment:
Does not fit the previous readme information

Reply:
Changed